### ISSUE TITLE
Error in ProxyAPI.Info

### ISSUE THREAD

**Issue by hferentschik** [fabric8-services/fabric8-jenkins-proxy#133]
When turning on debug trace, we get a whole bunch of:

    {"level":"debug","msg":"Did not find data for none","time":"2018-02-09T11:00:21Z"}

Looking at the underlying code I am also wondering a bit about the implementation:

```Go
func (api *ProxyAPI) Info(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
	ns := ps.ByName("namespace")
	s, notFound, err := api.storageService.GetStatisticsUser(ns)
	if err != nil {
		if notFound {
			log.Debugf("Did not find data for %s", ns)
		} else {
			log.Error(err) //FIXME
			fmt.Fprintf(w, "{'error': '%s'}", err)
			return
		}
	}
	c, err := api.storageService.GetRequestsCount(ns)
	if err != nil {
		log.Error(err) //FIXME
		fmt.Fprintf(w, "{'error': '%s'}", err)
		return
	}
	resp := APIResponse{
		Namespace:   ns,
		Requests:    c,
		LastRequest: s.LastBufferedRequest,
		LastVisit:   s.LastAccessed,
	}

	json.NewEncoder(w).Encode(resp)
}
```

If there is no record found, will there be also an error? That seems odd.

**Comment by hferentschik** [comment]
Also where are these request for 'none' coming from?

**Comment by vpavlin** [comment]
I don't see this error in logs anymore. Let's watch Kibana for a little while and see if it comes back

**Comment by vpavlin** [comment]
https://logs.rh-idev.openshift.com/goto/4667234eb42a271c2d15c78062cb353d

**Comment by kishansagathiya** [comment]
![screenshot from 2018-03-22 20-45-05](https://user-images.githubusercontent.com/10185999/37779305-1f109d68-2e12-11e8-9c3d-8952ac40f4fb.png)

Checkout Readiness probe and Liveness probe. These requests are coming from Openshift.
@chmouel WDYT should I close this issue?

**Comment by chmouel** [comment]
I think the idea here is to not have that error in the log, should we discard it if you think those are normal ?

**Comment by kishansagathiya** [comment]
Yeah, you are right. What the issue really about is that whether there can be a situation when it is found and we still get an error. If not, it will never reach that else block and log the error. 
I think we dont't need to change anything here. It is a safe implementation since we are having two confirmation here. We can close this. WDYT?

**Comment by chmouel** [comment]
But those logs should be discarded i think since they just provide nose : 

```
{"level":"debug","msg":"Did not find data for none","time":"2018-02-09T11:00:21Z"}
```

**Comment by kishansagathiya** [comment]
Cool. If we get rid of that debug logs here, we can move error log out of else block. That will actually give better info. I will get this done.

**Comment by kishansagathiya** [comment]
> If there is no record found, will there be also an error? That seems odd.

Yes, https://github.com/jinzhu/gorm/blob/master/main.go#L519

@chmouel We can get rid of these logs also storageService.GetStatisticsUser doesn't need to return notFound. I see other such error handlings using storageService.GetStatisticsUser as well. One here https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/internal/proxy/proxy.go#L615-L624

**Comment by kishansagathiya** [comment]
I have a feeling that removing showing not found as an error would fails Liveliness and Readiess Probe, right?
